### ISSUE TITLE
isFrozen always return true?

### ISSUE THREAD

**Issue by jeancolasp** [alembic/alembic#394]
I have been compiling the code recently and tried some basic examples. For a strange reason the `isFrozen` method always returns true (from lib/Alembic/Ogawa/IStreams.cpp). I tried several files that were all generated from Maya (2022). Looking at the code, I tried to replicate this by just reading the first 16 chars from the file and testing the value of the 6th one (header[5]). The test (header[5] == char(0xff)) is always true.

What can I be missing? I'll try to find ABC files coming from other sources, but I find it strange that Maya would generate unproper files. If you have any idea I'd greatly appreciate your help (I am on Windows 10).

**Comment by jeancolasp** [comment]
It seems like isFrozen 'is' supposed to always return true is the file was actually successfully written to disk (looking at Alembic/Ogawa/OStream.cpp line #121). But it someone could confirm I would appreciate it. Thank you.

**Comment by lamiller0** [comment]
This is correct.  This is a helpful indicator that the file properly finished writing, either because something is still writing to it, or the DCC spectacularly crashed in the middle of writing data.
